salesforce/backend/query.py

```diff
@@ -483,6 +513,12 @@ def execute(self, q, args=()):
             self.results = iter([])
 
     def execute_select(self, q, args):
+        if q == "SELECT django_migrations.app, django_migrations.name FROM django_migrations":
```

Comment:
These are very interesting insights, thank you! Regarding faster tests / development with SQLite as a test backend for SF: we were using it heavily in the beginning, but came to the conclusion, that it is not a good replacement. SF itself implements so much logic, that cannot be easily "simulated" with a normal DB. This does not even include special `Flow` events.